src/store/Modules/Projects.ts

```diff
@@ -529,15 +529,18 @@ export default class Projects extends VuexModule {
         },
       }
     })
-    if (projects.length > 0) {
-      this.context.commit("addInitialProjects", data)
-    } else {
-      this.context.commit("openSampleDataPopup")
+    try {
+      if (projects.length > 0) {
+        this.context.commit("addInitialProjects", data)
+      }
+    } catch (err) {
+      console.log(err)
+      this.context.commit("addInitialProjects", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.